### PR TITLE
add `else` to docs

### PR DESCRIPTION
**Pull request by justjake**
@hshoff @horaceko

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -637,6 +637,29 @@
...
+    }
+    else {
+      thing3();
+    }
+
+    // good
+    if (test) {
+      thing1();
+      thing2();
+    } else {
```
Comment (horaceko) [inline README.md]: What! WHY. I much prefer the former.

**Comment by ascott** [comment]
LGTM

**Comment by emroot** [comment]
LGTM

**Comment by goatslacker** [comment]
LGTM

**Pull request merged**